index.js

```diff
@@ -107,9 +109,54 @@ app.delete('/ideas', (req, res) => {
     res.status(200).send('Ideas successfully deleted.');
 });
 
+//USER METHODS
+app.get('/users', (req, res) => {
+    res.status(200).json(getUsers());
+});
+
+app.post('/users', (req, res) => {
+    try {
+        registerUser(
+            req.body.userName,
+            req.body.password,
+            req.body.firstName,
+            req.body.lastName
+        );
+    } catch (error) {
+        res.status(400).send(error.message);
+    }
+    res.status(200).send('User created successfully.');
+});
+
+app.put('/users', (req, res) => {
+    try {
+        updateUser(req.body.userId, req.body.key, req.body.value);
+    } catch (error) {
+        res.status(400).send(error.message);
+    }
+    res.status(200).send('User successfully update.');
+});
+
 //Login
 
 app.post('/login', (req, res) => {
+
+    try {
+        login(req, res);
```

Comment:
Why does the `login` function need `req` and `res` objects as an input?

Reply:
Updated and tested.